client/src/templates/Challenges/utils/index.js

```diff
@@ -25,3 +25,7 @@ export function transformEditorLink(url) {
       '//glitch.com/edit/#!/$<projectname>'
     );
 }
+
+export function removeUrlParameters(url) {
+  return url.replace(/\?.+/, '');
+}
```

Comment:
```suggestion
export function removeUrlParameters(url) {
  return url.replace(/\?.+/, '');
}
```
Rather than using a regex, we can just use the [URL api](https://developer.mozilla.org/en-US/docs/Web/API/URL) for this. It'll be a bit more verbose, but should be more robust.

Reply:
You see, I would have thought about doing this, but I didn't...